### DISCUSSION TITLE
on windows 7 32

### DISCUSSION THREAD

**Discussion by arielextreme** [diasurgical/DevilutionX#6959] [General]
This warning appears when I run "devilutionx.exe" how to fix it, my Windows 7 is 32-bit 

![0cd0b9e1-6a39-4add-a83b-f8145636a00d](https://github.com/diasurgical/devilutionX/assets/83328460/5aca6f1c-5966-4f5d-8517-11eadcc87a07)

**Comment by DakkJaniels** [comment]
Did you try using the [devilutionx-windows-i386.zip](https://github.com/diasurgical/devilutionX/releases/download/1.5.2/devilutionx-windows-i386.zip) version or the [devilutionx-windows-x86_64.zip](https://github.com/diasurgical/devilutionX/releases/download/1.5.2/devilutionx-windows-x86_64.zip) version?

**Reply by arielextreme** [reply]
Thanks it works

**Comment by AJenbo** [comment]
Which version did you download? If it was the x86_64 then it's the 64bit version which you can't run on your 32bit system. Download the i386 on instead (like Dakkjaniels suggested), it's for 32bit computers.

**Reply by arielextreme** [reply]
Thanks it works
